pkg/client/cmdline.go

```diff
@@ -12,21 +12,32 @@ import (
 	"strings"
 )
 
-// Bootloader determines which kernel/initrd/kernel args to use when starting
-// the virtual machine. It is mandatory to set a Bootloader or the virtual
+// Bootloader is the base interface for all bootloader classes. It specifies how to
+// boot the virtual machine. It is mandatory to set a Bootloader or the virtual
 // machine won't start.
-type Bootloader struct {
+type Bootloader VMComponent
+
+// linuxBootloader determines which kernel/initrd/kernel args to use when starting
+// the virtual machine.
+type linuxBootloader struct {
 	vmlinuzPath   string
 	kernelCmdLine string
 	initrdPath    string
 }
 
+// efiBootloader determines which kernel/initrd/kernel args to use when starting
+// the virtual machine.
```

Comment:
Looks like it is copy/paste from LinuxBootloader.

Reply:
Thanks, fixed.